internal/cephfs/core/volume.go

```diff
@@ -199,16 +199,13 @@ func (s *subVolumeClient) GetSubVolumeInfo(ctx context.Context) (*Subvolume, err
 type operationState int64
 
 const (
-	unknown operationState = iota
-	supported
+	supported operationState = iota
 	unsupported
 )
 
 type localClusterState struct {
```

Comment:
`localClusterState` might need to be removed as well as we don't require it anymore.

Reply:
Since the checks are needed, keeping it as it is for the time being.